File: python_programming_basics/while_loops/l_05_account_balance.py
command = input()
total_sum = 0

while command != "NoMoreMoney":
    increase_sum = float(command)

    if increase_sum < 0:
        print("Invalid operation!")
        break

    else:
        print(f"Increase: {increase_sum:.2f}")
        total_sum += increase_sum

    command = input()

print(f"Total: {total_sum:.2f}")

File: JavaScript Basics/05. While Loop/l-05-Account-Balance.js
function calculateAccount(input) {
    let index = 0;
    let total = 0;

    while (input[index] !== "NoMoreMoney") {
        let amount = parseFloat(input[index]);
        if (amount < 0) {
            console.log("Invalid operation!");
            break;
        }

        console.log(`Increase: ${amount.toFixed(2)}`);
        total += amount;
        index++;
    }

    console.log(`Total: ${total.toFixed(2)}`);
}

calculateAccount(["5.51", "69.42", "100", "NoMoreMoney"]);
calculateAccount(["120", "45.55", "-150"]);
